src/Ryujinx.Common/Configuration/AppDataManager.cs

```diff
@@ -134,6 +118,62 @@ private static bool IsPathSymlink(string path)
             return (attributes & FileAttributes.ReparsePoint) == FileAttributes.ReparsePoint;
         }
 
+        public static void FixMacOSConfigurationFolders()
+        {
+            if (OperatingSystem.IsMacOS())
+            {
```

Comment:
```suggestion
        [SupportedOSPlatform("macos")]
        public static void FixMacOSConfigurationFolders()
        {
```
(Make sure to remove the if statement manually and not using the suggestion here.)